vnext/Desktop.IntegrationTests/WebSocketResourcePerformanceTests.cpp

```diff
@@ -53,27 +56,52 @@ TEST_CLASS (WebSocketResourcePerformanceTest) { // See http://msdn.microsoft.com
     const int startThreadCount = GetCurrentThreadCount();
 
     std::atomic_int32_t threadCount = 0;
-
+    bool errorFound = false;
+    string errorMessage;
+
+    // TODO: Re-enable.
+    // auto server = std::make_shared<Test::WebSocketServer>(5556);
+    // server->SetMessageFactory([](string&& message)
+    //{
+    //  return message + "_response";
+    //});
+    // server->Start();
     // WebSocket resources scope.
     {
-      vector<unique_ptr<IWebSocketResource>> resources;
+      vector<shared_ptr<IWebSocketResource>> resources;
       for (int i = 0; i < resourceTotal; i++) {
         auto ws = IWebSocketResource::Make("ws://localhost:5555/");
```

Comment:
>5555 [](start = 59, length = 4)

Julio, can you please start new branch or fork. It's hard to keep track of the commits here.